pages/levels/freeplay/index.js

```diff
@@ -44,14 +44,10 @@ export default function Level2() {
     setTextState(0);
     setChooseTileState(chooseTileState + 1);
   }
-  //---------------Winning----------------------------------
-  if (array.some(item => item.color === "lumberhut")) {
-    setTimeout(() => setOverlayState("win"), 1500);
-  }
```

Comment:
why are you deleting this? :)

Reply:
The freeplay shouldn't have a winning condition. that one was still there from my copy paste of level 1